src/scores/probability/crps_impl.py

```diff
@@ -174,35 +174,39 @@ def crps_cdf(
     is used as an index for predictive CDF values. Various techniques are used to
     interpolate CDF values between indexed thresholds.
 
-    Given
-        - a predictive CDF `fcst` indexed at thresholds by variable x,
-        - an observation in CDF form `obs_cdf` (i.e., obs_cdf(x) = 0 if x < obs and 1 if x >= obs),
-        - a `threshold_weight` array indexed by variable x,
+    Given:
+        - a predictive CDF `fcst` indexed at thresholds by variable x
+        - an observation in CDF form `obs_cdf` (i.e., :math:`obs\\_cdf_x = 0` if :math:`x < obs` and 1 if :math:`x >= obs`)
```

Comment:
There needs to be a couple of changes to this.
1. use `\text{}` around variables with names
2. Instead of `_x`, replace it with `(x)`

Reply:
Thanks, that was an oversight.